spec/unit/coinbase/address_reputation_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+describe Coinbase::AddressReputation do
+  let(:address_id) { '0x123456789abcdef' }
+  let(:network) { 'ethereum-mainnet' }
+  let(:mock_model) do
+    instance_double(
+      Coinbase::Client::AddressReputation,
```

Comment:
We should add a factory for `AddressReputation`: https://github.com/coinbase/coinbase-sdk-ruby/blob/master/spec/factories/fund_quote.rb